themes/src/main/resources/theme/keycloak/welcome/theme.properties

```diff
@@ -1,7 +1,7 @@
-styles=css/welcome.css
 import=common/keycloak
```

Comment:
Do we still need this?

Reply:
I think that any common asset for the Welcome page should be considered a global resource since does not belong to a realm or a theme.  The Welcome `theme type` is an odd duck and is actually global in and of itself.  I suspect its resources should be handled the same way we did the common React library?

It's a little weird that these global resources now live in the legacy `keycloak` theme, which is destined to be deprecated.  If it's easy to fix this in this PR then let's go ahead and do it.  Otherwise, please create an issue for it and add to the issues list for the "commons" epic, https://github.com/keycloak/keycloak/issues/25552.

Reply:
> I think that any common asset for the Welcome page should be considered a global resource since does not belong to a realm or a theme.

It might not depend on a realm, but it is technically part of a theme, specifically `keycloak`. I am not entirely sure what a 'global resource' would mean here.

> The Welcome theme type is an odd duck and is actually global in and of itself. I suspect its resources should be handled the same way we did the common React library?

Yeah, it's a bit of a strange one, it still always belongs to a theme though, so I think it's fine to keep it's shared resources in the theme itself.

> It's a little weird that these global resources now live in the legacy keycloak theme, which is destined to be deprecated.

I don't think we'll be getting rid of the `keycloak` theme anytime soon, as we still have the `email` and `login` theme types in there. I'd actually propose we merge all the `keycloak.vX` themes back into `keycloak`, as we no longer have code for the original themes around in the long run, just adding confusion.